### ISSUE TITLE
Q: Take out the Garbage: Bonuses

### ISSUE THREAD

**Issue by awesomebytes** [RoboCupAtHome/RuleBook#658]
Hello,

The general scoring rules (3.8.2.4) state: `No bonus points can be scored when the robot requests help to solve part of a task that normally would grant a bonus.`

Take out the Garbage states a Bonus rewards (2) as `Transporting both bags at once. (300pts)`.

Hereby I'm mostly asking for SSPL, given that picking up the trash bag from a bin seems unrealistic (or impossible) for the Pepper platform, the `Deus ex Machina` rule about score reduction: `Score reduction for requesting human assistance is applied per bag as follows: * Handing a bag over to the robot results in a score reduction of 200pts per bag.` Will most certainly apply for both bags.

As the robot has to ask for help to pick up the trash bag (so we trigger a Deus ex Machina rule) does this mean the Bonus reward is unattainable for this league?

What if we ask a human to place the trash bag in a graspable location for Pepper?

I'm assuming (as discussed here https://github.com/RoboCupAtHome/RuleBook/issues/575) that we won't have the trash bag placed in a trash bin at a special height for Pepper.

Thanks.

**Comment by kyordhel** [comment]
## TL; DR: No

The point of the test is to have a robot taking out the garbage **autonomously**.

Pepper has two arms while the HSR does not, so we would be giving SSPL 200pts for free. if we are going to allow handing over bags, it must be approved for all 3 leagues. Such modification would mean allowing people to hang N bags on the robot’s gripper, making the bonus trivial in DSPL and OPL. 

INSTEAD, I’d like to see Pepper try to autonomously solve the task with the HW it has. If this requires carrying one bag with both hands, so be it.

The bonus for carrying two bags remains as is since it involves more planning than manipulation.

**Comment by awesomebytes** [comment]
We'd love to make Pepper grab both bags.
The issue is about the robot only being able to grasp on a specific height
margin. The arms are short and the crouching of the robot is very limited.
It's like a grandpa T-Rex that cant reach his feet anymore. Typical
trashbins (including the ones in the discussion in the test creation) are
all way too short for the robot to pick up the trash bag.

It was already discussed that no special furniture / objects will be used
for SSPL. Being excluded that, we believe the bonus just can't be applied
for SSPL.

To avoid that kinda sad fact, the "exception" we can think of would be if
it was agreed to place the trashbins (or the trash bags) on a reachable
height for the robot. Just put a box or a bunch of books under the bin (or
on the bottom of the bin if it has a usable size to make just the bag be at
a reachable height).

It may sound cumbersome but we'd still be using the same furniture/objects
and at least allow the SSPL league to try to achieve the bonus scores
(maybe taking the lid off too?).

Thanks!

**Comment by kyordhel** [comment]
I can neither agree nor disagree since I don't know how the bins will look like. I'm pretty sure the TC will think about a solution so the bins are within grasping distance. However, this solution depends on the available furniture. Considering the limited budget, I can't promise anything.

I think the height of the bins should be discussed during the team leader meeting.

**Comment by awesomebytes** [comment]
Thanks for the reply. As a TC I'll try to make a reasonable tradeoff to have the bin in a suitable height for the SSPL league.
